frontend/src/components/submission/SubmissionForm.vue

```diff
@@ -88,11 +90,22 @@ const formSchema = object({
     .label('Submission Types'),
   user: mixed()
     .when('intakeStatus', {
-      is: (val: string) => val !== 'SUBMITTED',
-      then: (schema) => schema.required(),
-      otherwise: (schema) => schema.notRequired()
+      is: (val: string) => val === INTAKE_STATUS_LIST.SUBMITTED,
+      then: (schema) =>
+        schema
+          .test('expect-user-or-empty', 'Assigned to must be empty or a selected user', (obj) => {
```

Comment:
you could rename `obj` to `input` to make it more readable:
`.test('expect-user-or-empty', 'Assigned to must be empty or a selected user', (input) => {`